airbyte-integrations/connectors/source-hubspot/source_hubspot/api.py

```diff
@@ -125,6 +125,8 @@ class API:
 
     def __init__(self, credentials: Mapping[str, Any]):
         self._credentials = {**credentials}
+        if "token_expires" in credentials:
```

Comment:
why not using cdk's oauth authenticator?

Reply:
We don't use CDK in this connector and the auth was previously implemented there. Of course, we could start rewriting of the connector by using the CDK authenticators. It might be an option there.